components/feedback/style.scss

```diff
@@ -39,7 +39,7 @@ $tna_outline: #fce45c;
   }
 }
 
-#details-feedback-wrapper {
+#details-feedback-wrapper, #what-to-expect {
```

Comment:
can have the id as `#what-to-expect-form`

Reply:
Done